docs/tutorials/interactive-terminal.md

```diff
@@ -0,0 +1,199 @@
+* [Adafruit FTDI Friend](https://www.adafruit.com/product/284)
+* [Sparkfun FTDI Cable 5V VCC-3.3V I/O](https://www.sparkfun.com/products/9717)
+* [Use an Arduino board as a serial adapter](arduino-serial-forwarder.html)
+
+In Norway you should be able to find them in any of these stores:
+* [Kjell & Company](https://www.kjell.com/no/sok?query=ftdi)
+* [Digital Impuls](https://www.digitalimpuls.no/search?q=ftdi)
+* [ELFA Distrelec](https://www.elfadistrelec.no/)
+
+### Install serial adapter driver
+Most cables require you to install a driver for the USB serial adapter. Check the product description and it should guide you to the correct driver for your operating system.
+
+### Serial terminal application
+You can use any serial terminal application you like. We'll only explain how to use CoolTerm in this guide - since it's free, easy to use and works on Windows, macOS and Linux.
+
+1. [Download and install CoolTerm for your OS](http://freeware.the-meiers.org/)
+1. Open app and click the «Options» button
+1. Choose which serial port to use
+    - This depends on which driver and operating system you use. 
+        - In Linux the serial ports are treated as files, typical examples would be `/dev/ttyS0`, `/dev/ttyUSB0` if you use a USB to serial adapter or `/dev/ttyAMA0` on the Raspberry Pi.
+        - In Mac OSX serial ports can have various names, for example `usbserial-FTACX955`.
+        - At last there is Windows. In Windows, the serial port is an abstraction called a "COM port". You can find these by looking the device manager. In the screenshot below, we have connected a TTL-232 USB cable to the machine:
+        ![Device manager screenshot](img/device-manager-arrow.png)
+1. Set the baud rate to 9600
+
+  ![CoolTerm Serial Port options screenshot](img/coolterm-serialport.png)
+1. Go to «Terminal» options (on the left)
+1. Change to «Line Mode»
+1. Check «Local Echo» to see the commands you've sent
+1. Leave the rest of the options as default and click «OK»
+
+  ![CoolTerm Terminal options screenshot](img/coolterm-terminal.png)
+1. Click the «Connect»-button
+1. Enter commands in the input field near the bottom of the window
+
+### Connecting the cables
+Normally on a serial connection, the TX is the output from one device and RX is the input. So when you connect two devices, you would connect TX->RX both ways. However, u-blox thought otherwise, so they've labled the input as TX and output as RX. To avoid an extra layer of confusion we used the same labeling as u-blox on the EE-NBIOT-01. So when connecting the wires - **connect RX to RX and TX to TX**.
+
+**NB! Make sure the cable has 5V to 3.3V logic level conversion on RX&TX**
+
+**Table 1:**
+
+Serial adapter | EE-NBIOT-01
+--------------:|------------
+VCC +5V/3.3V - red | VCC
+GND - black | GND
+RX - yellow | RXD
+TX (**3.3V**) - orange | TXD
+
+[![FTDI to EE-NBIOT-01](img/ftdi-small.jpg)<br/>
+See image in full resolution](img/ftdi.jpg)
+
+## Terminal commands
```

Comment:
Can you add an anchor here?  The Raspberry Pi serial tutorial wants to link to this.

Reply:
Call it `terminal-commands` if you would be so kind.

Reply:
That's done automagically from the title ;)
[link](http://localhost:8888/tutorials/interactive-terminal.html#terminal-commands)

Reply:
Nice :)  Too bad you can't see the anchor when you hover..

Reply:
True, but viewing the source and checking for an id attribute always works.

Reply:
It's just nice for normal users (peasants) to be able to link to sections.